### PR TITLE
fix: default project banner

### PR DESCRIPTION
**Pull request by sk-tenba**

### CODE REVIEW

**Comment by sk-tenba** [comment]
image

File: modules/teiki-components/components/PanelProjectOverviewV2/components/BannerCarousel/index.tsx [hunk 1]
```diff
@@ -65,7 +66,13 @@ export default function BannerCarousel({
...
                   crop={items[index].crop}
                 />
-              ) : null}
+              ) : (
+                <ImageView
+                  style={{ width: "100%", height: "100%" }}
+                  src={DEFAULT_IMAGE_URL}
+                  crop={{ x: 0, y: 0, w: 1, h: 1 }}
+                />
+              )}
```
Comment (sk-kitsune) [inline modules/teiki-components/components/PanelProjectOverviewV2/components/BannerCarousel/index.tsx:75]: A bit opinionated but I want to use the default image only if the length is 0

Note that `numBoxes` can be set to _any_ integer larger than `items.length`, although I did not document it.

Suggestion: change from `<ImageView />` to `!index ? <ImageView ... /> : null`

File: modules/teiki-components/components/PanelProjectOverviewV2/components/BannerCarousel/index.tsx [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import cx from "classnames";
 import * as React from "react";
 
+import { DEFAULT_IMAGE_URL } from "../../../../../../containers/PageHome/components/ProjectImageCropped/constants";
```
Comment (sk-kitsune) [inline modules/teiki-components/components/PanelProjectOverviewV2/components/BannerCarousel/index.tsx:4]: please copy this const to this folder

---

🔴 **Blocking issue**: this PR cannot be merged until this issue is resolved.

**Review by sk-kitsune — CHANGES_REQUESTED** [review]
⛔ **Changes requested**: there are blocking issues.

**Review by sk-kitsune — APPROVED** [review]
✅ **Approved**: no blocking issues

**Pull request merged**